### ISSUE TITLE
Feature Request: Nvidia driver rollback

### ISSUE THREAD

**Issue by alexjfinch** [Nobara-Project/nobara-core-packages#20]
I've been having trouble recently that has the potential to be linked to the following on Nvidia reporting;

https://forums.developer.nvidia.com/t/555-crashes-xwayland-when-playing-games/298306

I can't even install games through steam at the moment, the whole system just locks out after a while. 

Given the installation script provided through the updater and driver manager does a lot of background additional checks, it would be helpful if a rollback feature could be incorporated where you could go back to the driver before that was stable or potentially you could pick a driver number to reinstall and troubleshoot issues with potential drivers.

**Comment by Ironlungs76** [comment]
@GloriousEggroll   Since the new update drivers of Nvidia 565.77, Assassin's Creed Valhalla hasn't been playable , their some artefacts glitches in the game making it unplayable ,GloriousEggroll do you think in the Nobora driver manager you could put mores drivers so people can downgrade or install the Latest Production Branch Version  (550.144.03)  

also some links to reddit showing people having the same issues as me
https://www.reddit.com/r/linux_gaming/comments/1gkxezi/graphics_issue_on_ac_valhalla/
![Image](https://github.com/user-attachments/assets/393bfb00-28ae-4923-8c9a-92ac05f653e1)

**Comment by Haxorzz** [comment]
> [@GloriousEggroll](https://github.com/GloriousEggroll) Since the new update drivers of Nvidia 565.77, Assassin's Creed Valhalla hasn't been playable , their some artefacts glitches in the game making it unplayable ,GloriousEggroll do you think in the Nobora driver manager you could put mores drivers so people can downgrade or install the Latest Production Branch Version (550.144.03)
> 
> also some links to reddit showing people having the same issues as me https://www.reddit.com/r/linux_gaming/comments/1gkxezi/graphics_issue_on_ac_valhalla/ ![Image](https://github.com/user-attachments/assets/393bfb00-28ae-4923-8c9a-92ac05f653e1)

just an FYI this was fixed on 570 via an application profile

**Comment by Ironlungs76** [comment]
> > [@GloriousEggroll](https://github.com/GloriousEggroll) Since the new update drivers of Nvidia 565.77, Assassin's Creed Valhalla hasn't been playable , their some artefacts glitches in the game making it unplayable ,GloriousEggroll do you think in the Nobora driver manager you could put mores drivers so people can downgrade or install the Latest Production Branch Version (550.144.03)
> > also some links to reddit showing people having the same issues as me https://www.reddit.com/r/linux_gaming/comments/1gkxezi/graphics_issue_on_ac_valhalla/ ![Image](https://github.com/user-attachments/assets/393bfb00-28ae-4923-8c9a-92ac05f653e1)
> 
> just an FYI this was fixed on 570 via an application profile

Really? is 570 out yet ?

**Comment by Haxorzz** [comment]
> > > [@GloriousEggroll](https://github.com/GloriousEggroll) Since the new update drivers of Nvidia 565.77, Assassin's Creed Valhalla hasn't been playable , their some artefacts glitches in the game making it unplayable ,GloriousEggroll do you think in the Nobora driver manager you could put mores drivers so people can downgrade or install the Latest Production Branch Version (550.144.03)
> > > also some links to reddit showing people having the same issues as me https://www.reddit.com/r/linux_gaming/comments/1gkxezi/graphics_issue_on_ac_valhalla/ ![Image](https://github.com/user-attachments/assets/393bfb00-28ae-4923-8c9a-92ac05f653e1)
> > 
> > 
> > just an FYI this was fixed on 570 via an application profile
> 
> Really? is 570 out yet ?

Yep. It's a stable new feature branch afaik, they fixed multi monitor VRR among other things

**Comment by GloriousEggroll** [comment]
We've implemented the ability to change between branches as well as between open+closed kernel module via point+click in our new driver manager update. It's not exactly a full rollback option but as close as it's going to get. Closing this as resolved
